src/main/java/org/libtorrent4j/SessionManager.java

```diff
@@ -478,7 +486,9 @@ public TorrentHandle find(Sha1Hash sha1) {
      * @param resumeFile the file with the resume file
      * @param priorities the initial file priorities
      */
-    public void download(TorrentInfo ti, File saveDir, File resumeFile, Priority[] priorities, List<TcpEndpoint> peers) {
+    public void download(TorrentInfo ti, File saveDir, File resumeFile,
```

Comment:
Codacy found an issue: [The method 'download(TorrentInfo, File, File, Priority, List, torrent_flags_t)' has an NPath complexity of 6400, current threshold is 200](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5876791)